Jenkinsfile.integration

```diff
@@ -23,7 +23,7 @@ pipeline {
     parameters {
         /* first value in the list is the default */
         choice(choices: ['airship', 'osh'], description: 'Which deployment mechanism?', name: 'deployment')
-        choice(choices: ['caasp3', 'caasp4'], description: 'Which caasp version to use?', name: 'caasp_version')
+        choice(choices: ['caasp4'], description: 'Which caasp version to use?', name: 'caasp_version')
```

Comment:
should be removed if we are going for caasp4 only

Reply:
I understood that because the second point in the card acceptance criteria is remove CaaSPv3. Is it correct? Has to be removed from CI too? @jhesketh

Reply:
well, if we remove caasp3 playbooks, then its impossible for us to test caasp3 in CI so it should also be dropped

Reply:
Yes, let's remove the choice completely